.github/workflows/image-push-master.yml

```diff
@@ -27,7 +30,98 @@ jobs:
         with:
           context: .
           push: true
+          platforms: linux/amd64
           tags: |
-            ${{ env.IMAGE_NAME }}:latest
-            ${{ env.IMAGE_NAME }}:${{ github.sha }}
+            ${{ env.IMAGE_NAME }}:latest-amd64
```

Comment:
can you keep ${{ env.IMAGE_NAME }}:${{ github.sha }} for amd64 images ?

Reply:
Sure, and is it just for amd64?

Reply:
yes, like it was before